Tests/eZ/Publish/FieldType/TweetTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
```

Comment:
Must be moved to another commit. I was thinking about merging it with the code it tests, but @mere-teresa suggested a distinct commit, therefore located right after the addition of the tested code.

In addition, this should contain some kind of instructions about running those tests. PhpDoc in the test classes would work for me.